src/Tabs/Arrow.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from "prop-types";
+import React, { useRef } from "react";
+import { CSSTransition } from "react-transition-group";
+import IconButton from "../IconButton";
+
+const Arrow = ({ direction, onClick, show, isResponsive }) => {
+  const arrowRef = useRef();
+
+  if (isResponsive === true) {
```

Comment:
You could do something like ```return (isResponsive && (```. The other return doesn't need to be here. We either want to have arrows reserve space or not have arrows at all.

One good rule of thumb is to use the context properties where they need to be used and not pass them down as props.